src/app/components/flyout/flyout-guidelines.component.html

```diff
@@ -0,0 +1,273 @@
+<section>
+  <header><div class="sky-section-heading">Usage</div></header>
+  <stache-row>
+    <stache-column screenSmall="6">
+			<p>A flyout is a container that slides out from the side of the page and lays over page content to display large amounts of supplementary information related to user tasks. A flyout can contain lists, record views, graphs, or bespoke information displays.</p>
```

Comment:
What are "bespoke information displays"? I don't know what that means, but I left it in place because I don't know how to edit it.